config/manager/manager.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: v1
 kind: Namespace
 metadata:
+  annotations:
+    workload.openshift.io/allowed: management
```

Comment:
I'm not sure how this will work when deployed by OLM, but adding a annotation won't hurt here